### PR TITLE
Add file upload option & password fields to settings

### PR DESCRIPTION
**Pull request by prat98**
This PR is for AAP-24760 and AAP-24670. It adds a password field to all secrets in the settings forms and adds fileupload forms to the SAML settings section

### CODE REVIEW

File: frontend/awx/administration/settings/AwxSettingsForm.tsx [hunk 1]
```diff
@@ -168,6 +169,38 @@ export function AwxSettingsForm(props: {
 
 export function OptionActionsFormInput(props: { name: string; option: AwxSettingsOptionsAction }) {
   const option = props.option;
+
+  if (option.label.endsWith('Secret')) {
```
Comment (nixocio) [inline frontend/awx/administration/settings/AwxSettingsForm.tsx:173]: I read this code quickly @prat98 would this label being translated?

Comment (prat98) [reply frontend/awx/administration/settings/AwxSettingsForm.tsx:173]: Hmm good question maybe I should just do `.contains(t('Secret'))`. That way I capture the word in the label regardless. Certain translations may update the order of the words.

Comment (nixocio) [reply frontend/awx/administration/settings/AwxSettingsForm.tsx:173]: I do not remember seeing such thing before. In general we might need to create an object or other data structure with an id that is not selected to be translated.

**Review by AlexSCorey — APPROVED** [review]
We spoke offline.  Looks good to me.!

File: frontend/awx/administration/settings/AwxSettingsForm.tsx (hunk 1 above)
Comment (prat98) [reply frontend/awx/administration/settings/AwxSettingsForm.tsx]: Hmm so what is your proposal?

**Pull request merged**